ingest/settings.py

```diff
@@ -0,0 +1,64 @@
+# File is responsible for defining globals and initializing them
+from mongo_connection import MongoConnection
+from bson.objectid import ObjectId
+
+MONGO_CONNECTION = MongoConnection()
+
+
+def init(study_id, study_file_id):
+    global study
+    study = Study(study_id)
+    global study_file
+    study_file = StudyFile(study_file_id)
+
+
+def get_study():
+    global study
+    return study
+
+
+def get_study_file():
+    global study_file
+    return study_file
+
+
+class Study:
+    STUDY_ACCESSION: str = None
+
+    def __init__(self, study_id):
```

Comment:
I think this will be cleaner if __init__ calls a method like 'fetch_study', that stores the study document, and then you just make getters get_accession, and get_file_type that read from the study object

Reply:
See [comment](https://github.com/broadinstitute/scp-ingest-pipeline/pull/163#discussion_r522325975)